full-stack-tests/presentation/src/backend/RulesetsEmbedding.test.ts

```diff
@@ -46,7 +39,13 @@ describe("RulesEmbedding", () => {
   });
 
   beforeEach(async () => {
-    imodel = createSnapshotFromSeed(testIModelName, "assets/datasets/Properties_60InstancesWithUrl2.ibim");
+    const iModelPath = path.resolve(__dirname, "RulesetEmbedding.test.bim");
+    fs.existsSync(iModelPath) && fs.unlinkSync(iModelPath);
```

Comment:
nit: is this really a conditional style we encourage?

Reply:
meh its in test code

Reply:
hence nit, resolve at will